HexCtrl/src/CHexCtrl.h

```diff
@@ -283,5 +285,6 @@ namespace HEXCTRL::INTERNAL
 		bool m_fKeyDownAtm { false };         //Whether some key is down/pressed at the moment.
 		bool m_fMenuCMD { false };            //Command to be executed through menu, not through key-shortcut.
 		bool m_fRedraw { true };              //Should WM_PAINT be handled or not.
+		wchar_t m_wcUnprintable { L'.' };     //Replacement character for unprintable characters

```

Comment:
Member variables better be grouped by type, for a least memory footprint possible.
Also `m_wchUnprintable` would be more conventionally correct.

Reply:
Renamed.
There are no other member variables of type wchar_t to group with.

Reply:
I was not clear enough, it's rather type size.